params/protocol_params.go

```diff
@@ -168,3 +198,29 @@ func (t CodeFormat) Validate() bool {
 	}
 	return false
 }
+
+func (t CodeFormat) CodeInfoFromRules(r Rules) CodeInfo {
+	return t.CodeInfoFromCodeFormat() | GenerateVmVersion(r).CodeInfoFromVmVersion()
+}
+
+// Supporting VmVersion
+type VmVersion CodeInfo
```

Comment:
Why are VmVersion and CodeFormat same `CodeInfo` type?
How about uint8?

Reply:
Thanks for the comment :) I reflected at https://github.com/klaytn/klaytn/pull/912/commits/f21a7293d0e66c538653f3713661fa1c0d7a6a49